go/api/command_options.go

```diff
@@ -321,3 +321,78 @@ func (opts *BaseScanOptions) toArgs() ([]string, error) {
 
 	return args, err
 }
+
+// Optional arguments to Restore(key string, ttl int64, value string, option *RestoreOptions)
+//
+// Note IDLETIME and FREQ modifiers cannot be set at the same time.
+//
+// [valkey.io]: https://valkey.io/commands/restore/
+type RestoreOptions struct {
+	// Subcommand string to replace existing key.
+	Replace string
+	// Subcommand string to represent absolute timestamp (in milliseconds) for TTL.
+	ABSTTL string
```

Comment:
A user can set any string for these 2 values causing command to fail. I think all fields should be private. To keep docs visible to you can move them to setters.